tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""INSPIRE module that adds more fun to the platform."""
+
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+from helpers.providers.faker import faker
+
+
+@pytest.fixture(scope="function")
+def generate_data_for_record():
+    def _generate_data_for_record(data=None, with_control_number=False):
```

Comment:
do we care to generate schema valid records?

Reply:
I refactor this part, now there is a provider to generate records, which is used everywhere. It's good to have an actual valid record

Reply:
added a validation step when you generate a record